### ISSUE TITLE
Updated BIDS convention

### ISSUE THREAD

**Issue by agahkarakuzu** [JosePMarques/MP2RAGE-related-scripts#9]
@marcelzwiers I saw that you wrote some BIDS wrappers for MP2RAGE data. Unfortunately the convention you described has undergone some changes as the extension proposal evolved. 

Data would look like:

```
└── sub-01/
     └── anat/
         ├── sub-01_inv-1_part-mag_MP2RAGE.nii.gz
         ├── sub-01_inv-1_part-phase_MP2RAGE.nii.gz
         ├── sub-01_inv-1_MP2RAGE.json
         ├── sub-01_inv-2_part-mag_MP2RAGE.nii.gz
         ├── sub-01_inv-2_part-phase_MP2RAGE.nii.gz
         ├── sub-01_UNIT1.nii.gz
         ├── sub-01_UNIT1.json
         └── sub-01_inv-2_MP2RAGE.json
````

The B1+ maps are going to have suffix `TB1map` and going to be accessible from the derivatives folder.

**Comment by marcelzwiers** [comment]
True, I noticed that too and even started to adapt the wrappers. But I paused that because the [BEP001](https://github.com/bids-standard/bids-specification/pull/508) proposal was retracted by you :-). I am happy to update my code if you can assure me a bit that the above structure is here to stay...

**Comment by agahkarakuzu** [comment]
@marcelzwiers majority of the proposal has been merged, will be a part of the release `1.5.0`, so they are here to stay :)

**Comment by marcelzwiers** [comment]
> The B1+ maps are going to have suffix `TB1map` and going to be accessible from the derivatives folder.

Thanks, I will make the changes, but `TB1map`?

**Comment by agahkarakuzu** [comment]
@marcelzwiers `TB1map` for transmit and `RB1map` for receive fields.

**Comment by marcelzwiers** [comment]
I finally came to update my BIDS wrappers to BIDS v1.5 (see #10)
